tariff/amber.go

```diff
@@ -87,7 +87,7 @@ func (t *Amber) run(done chan error) {
 		var res []amber.PriceInfo
 
 		if err := backoff.Retry(func() error {
-			return backoffPermanentError(t.GetJSON(t.uri, &res))
+			return request.BackoffDefaultHttpStatusCodesPermanently()(t.GetJSON(t.uri, &res))
```

Comment:
I'm not sure I follow. At least HTTP 500 is a retryable error.

Reply:
What makes 500 retryable? The original condition was `code >= 400 && code <= 599`, I just took that over.
Or do you mean 500 should be excluded from the defaults?

Reply:
I am not saying they should not be retried. I just took over the existing list. The reason for retrying a 429 is that we are doing exponential backoffs, thus adhering to servers request.

Reply:
I dont think we are as long as the rate limit parameters are being ignored.